Dockerfile

```diff
@@ -30,6 +30,10 @@ RUN yarn build
 
 FROM ${GO_IMAGE} as go-builder
 
+ARG GO_BUILD_TAGS="oss"
+ARG WIRE_TAGS="oss"
+ARG BINGO="true"
```

Comment:
BINGO is set to false on the local builds, [for example in make build-docker-full](https://github.com/grafana/grafana/pull/63893/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R178).

On builds in Drone or in our pipeline it will be set to true by default.